java/panama-bindings/generate-bindings.sh

```diff
@@ -5,21 +5,15 @@ set -e -u -o pipefail
 echo "Starting Panama FFM API bindings generation ..."
 REPODIR=$(cd "$(dirname $0)"; cd ../../ ; pwd)
 CURDIR=$(cd "$(dirname $0)"; pwd)
-CUDA_HOME=$(which nvcc | cut -d/ -f-4)
 TARGET_PACKAGE="com.nvidia.cuvs.internal.panama"
 
-# Try to verify that a include directory exists inside CUDA_HOME
-if [ ! -d "$CUDA_HOME/include" ];
-then
-  echo "$CUDA_HOME/include does not exist."
-  if [ -d "/usr/local/cuda/include" ];
-  then
-    echo "Setting CUDA_HOME to /usr/local/cuda"
-    CUDA_HOME=/usr/local/cuda
-  else
-    echo "Couldn't find a suitable CUDA include directory."
-    exit 1
-  fi
+if [ -n "${CONDA_PREFIX:-}" ] && [ -d "${CONDA_PREFIX}/targets/x86_64-linux/include" ]; then
+  CUDA_INCLUDE_DIR="${CONDA_PREFIX}/targets/x86_64-linux/include"
+elif [ -d "/usr/local/cuda/targets/x86_64-linux/include" ]; then
+  CUDA_INCLUDE_DIR="/usr/local/cuda/targets/x86_64-linux/include"
+else
+  echo "Couldn't find a suitable CUDA include directory."
+  exit 1
 fi
```

Comment:
```suggestion
TARGET_DIR="targets/x86_64-linux/include"
if [ -n "${CONDA_PREFIX:-}" ] && [ -d "${CONDA_PREFIX}/${TARGET_DIR}" ]; then
  CUDA_INCLUDE_DIR="${CONDA_PREFIX}/${TARGET_DIR}"
elif [ -d "/usr/local/cuda/${TARGET_DIR}" ]; then
  CUDA_INCLUDE_DIR="/usr/local/cuda/${TARGET_DIR}"
else
  echo "Couldn't find a suitable CUDA include directory."
  exit 1
fi
```

Would you consider centralizing the repeat part of the path (the way the CUDA headers are layed out) into a variable, as suggested here? That'd make this a bit easier to read, in my opinion.

It'd also reduce the effort required if/when we add `aarch64` support for the Java bindings.

Reply:
fixed